stories/ClaimCard.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Story, Meta } from '@storybook/react'
+import { withNextRouter } from 'storybook-addon-next-router'
```

Comment:
You can also remove this import from `stories/Main.stories.tsx`, since we removed the router from that component